src/Widget/Conversation/Messages/Message/Message.tsx

```diff
@@ -42,15 +43,17 @@ function Message({ messageConfig, animation }: Props) {
   const renderLottieContainer = () => {
     return (
       <AnimationContainer className="rcw-animation-container">
-        {animation && <Lottie animationData={animation} />}
+        {animation && animationIsVisible && <Lottie animationData={animation} />}
```

Comment:
Is it possible that add/removing `<Lottie />` element will mess with the message height?